src/Drivers/Gd/Driver.php

```diff
@@ -36,7 +36,7 @@ public function id(): string
     public function checkHealth(): void
     {
         if (!extension_loaded('gd') || !function_exists('gd_info')) {
-            throw new RuntimeException(
+            throw new DriverException(
```

Comment:
I think it's better having a specific exception for such error, this would allows for example to ignore the check of this exception for phpstan or allow the user to have a specific behavior for them.

It BC because DriverException extends RuntimeException.